docs/learn/learn-proxies.md

```diff
@@ -179,19 +202,25 @@ where you will be able to select the type of proxy for that specific account.
 
 ![proxy generation part 2](../assets/polkadot_add_another_proxy.png)
 
-> NOTE: You cannot create an anonymous function from the Accounts page, you must be on the
-> Extrinsics page.
+:::info Creating an anonymous function 
+
+You cannot create an anonymous function from the Accounts page, you must be on the
+Extrinsics page.
+
+:::
 
 ### Removing Proxies
 
+:::warning There is no way to get access to the proxy after deleting it
```

Comment:
Not true. It applies only for anonymous proxy. You can remove a staking proxy and still have access to that staking proxy account.